modules/claims_api/lib/claims_api/claim_logger.rb

```diff
@@ -41,6 +41,7 @@ def self.format_msg(tag, **params)
         msg.append("ITF: #{params[:detail]}") if params[:detail].present?
       when 'validate_identifiers'
         msg.append("RID: #{params[:rid]}") if params[:rid].present?
+        msg.append("ICN: #{params[:icn]}") unless params[:icn].nil?
```

Comment:
I would stick with the `if .present?` to keep consistency with the other `appends`.

Also, this switch statement won't get hit because we modified the `tag` value coming in to `validate_identifiers_after_MPI_ADD_REPSONSE`